src/calculateRentalCost.js

```diff
@@ -3,8 +3,25 @@
  *
  * @return {number}
  */
+const RENT_PRICE = 40;
+const LONG_TERM = 7;
+const LONG_TERM_DISCOUNT = 50;
+const MEDIUM_TERM = 3;
+const MEDIUM_TERM_DISCOUNT = 20;
+
 function calculateRentalCost(days) {
-  // write code here
+  const numberOfDays = days;
```

Comment:
The variable `numberOfDays` is redundant since `days` can be used directly. It's best to remove this unnecessary variable to simplify the code.